src/client/utils/hooks/useBreakpoint.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react'
+import { BREAKPOINTS } from '../mediaQueries'
+
+export function useBreakpoint() {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth)
+
+  useEffect(() => {
+    const onWindowResize = () => {
+      setWindowWidth(window.innerWidth)
+    }
+
+    window.addEventListener('resize', onWindowResize)
+
+    return () => window.removeEventListener('resize', onWindowResize)
+  })
+
+  return {
+    isLargeScreen: windowWidth >= BREAKPOINTS.largeScreen,
```

Comment:
nice! So we add more attributes here 👌 👍 when we need them

Reply:
Yes that was the idea! 😊

Reply:
Then it's more clearly tightly connected to the breakpoints we define in the `mediaQueries` file. Or that's at least what my thought was.